lib/schema/checks.go

```diff
@@ -138,6 +138,7 @@ func ValidateRequirements(reqs Requirements, stateDir string) (failed []*pb.Prob
 				WillBeCreated:     utils.BoolValue(vol.CreateIfMissing),
 				Filesystems:       vol.Filesystems,
 				MinFreeBytes:      vol.Capacity.Bytes(),
+				HighWatermark:     0, // skip disk usage check
```

Comment:
Hmm, not sure how this helps? If HighWatermark is not initialized, its value will also be 0, right?